samples/crypto/psa_tls/README.rst

```diff
@@ -29,6 +29,217 @@ Overview
 The sample can act as either a network server or a network client.
 By default, the sample is configured to act as a server.
 
+TLS version support
+===================
+
+The sample supports both TLS v1.2 and TLS v1.3.
+See the following table for the support overview for each compatible board target.
+The table mirrors the test setup used for TLS verification.
+
+.. tabs::
+
+   .. tab:: TLS v1.2
+
+      .. list-table::
+         :header-rows: 1
+         :widths: auto
+
+         * - Board target
+           - Backend
+           - CMSE supported
+           - DTLS supported
+           - Limitations / not supported
+           - Comments
+         * - ``nrf52840dk/nrf52840``
+             ``nrf9160dk/nrf9160``
+             ``nrf9151dk/nrf9151``
+           - :ref:`Mbed TLS legacy c33xx backend <nrf_security_backends_cc3xx>`
+           - No
+           - No
+           - AES256, AES-GCM, SHA-512
+           - Also supports :ref:`PSA Crypto nrf_oberon driver<nrf_security_drivers_oberon>`
```

Comment:
This should be the "Mbed TLS legacy nrf_oberon" backend.

Mbed TLS "legacy crypto" and "PSA crypto" are two distinct crypto APIs. In this row we are referring TLS using the "legacy crypto" APIs.

Both "legacy" and PSA crypto APIs have A HW driver (using cc3xx driver) and a SW implementation (using the nrf_oberon oberon), however their configuration and usage are distinct.

Reply:
Actually, I now see that the legacy crypto using "nrf_oberon" is listed as a separate row below.
So just remove the "Also supports: PSA Crypto nrf_oberon driver" comment here